src/api/ApiIndex.vue

```diff
@@ -95,7 +78,7 @@ function slugify(text: string): string {
           <h3>{{ item.text }}</h3>
           <ul>
             <li v-for="h of item.headers" :key="h.anchor">
-              <a :href="item.link + '.html#' + slugify(h.anchor)">{{ h.anchor }}</a>
+              <a :href="item.link + '.html#' + h.anchor">{{ h.text }}</a>
```

Comment:
Changing `h.anchor` to `h.text` is the only part of this PR that is strictly necessary to fix the heading text. However...

This code has been through a few previous iterations and, in my opinion, the call to `slugify()` no longer makes sense. The anchor should already be suitably slugified. The other changes are primarily aimed at ensuring that `anchor` is already in the correct form to be used without needing further manipulation.